README.rst

```diff
@@ -106,6 +106,18 @@ Development
 🤯
 
 
+Tests
+----------
+
+To run the projet unit tests, use:
+
+```
+make test
+```
```

Comment:
Should we add the note about how to run the enron_dataset_test?

Reply:
Yes! Added